src/modules/accounting/accounting.py

```diff
@@ -183,39 +190,33 @@ def _get_finalization_shares_rate(self, blockstamp: ReferenceBlockStamp) -> int:
         logger.info({'msg': 'Calculate shares rate.', 'value': shares_rate})
         return shares_rate
 
-    def get_rebase_after_report(self, blockstamp: ReferenceBlockStamp, cl_only=False) -> LidoReportRebase:
-        chain_conf = self.get_chain_config(blockstamp)
-        frame_config = self.get_frame_config(blockstamp)
-
-        last_ref_slot = self.report_contract.functions.getLastProcessingRefSlot().call(
-            block_identifier=blockstamp.block_hash,
-        )
-
-        if last_ref_slot:
-            slots_elapsed = blockstamp.ref_slot - last_ref_slot
-        else:
-            slots_elapsed = blockstamp.ref_slot - frame_config.initial_epoch * chain_conf.slots_per_epoch
-
+    def get_rebase_after_report(
+        self,
+        blockstamp: ReferenceBlockStamp,
+        ignore_execution_rewards: bool = False,
+    ) -> LidoReportRebase:
+        """
+        To calculate how much withdrawal request protocol can finalize - needs finalization share rate after this report
+        """
         validators_count, cl_balance = self._get_consensus_lido_state(blockstamp)
 
+        chain_conf = self.get_chain_config(blockstamp)
         timestamp = chain_conf.genesis_time + blockstamp.ref_slot * chain_conf.seconds_per_slot
```

Comment:
maybe it's better to introduce a method for this `get_timestamp_by_slot(slot, chain_conf)`

Reply:
Fixed